src/View/layout.html.twig

```diff
@@ -23,18 +23,17 @@
 
 	</head>
 <body>
-
     {% block header %} 
-    {% include 'Components/_header.html.twig' %}
+    	{% include 'Components/_header.html.twig' %}
```

Comment:
if your header and footer is available in all pages, you should put it ahead of header / footer block